functions/task_5.py

```diff
@@ -0,0 +1,8 @@
+# Вычисление квадрата числа с помощью lambda
+# lambda-функция — это анонимная (безымянная) функция, которая используется для коротких вычислений
+def square(x):
+    return x ** 2
```

Comment:
square is not a lambda function)
here how it should look like

```suggestion
square = lambda x: x ** 2
```